src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -127,7 +127,7 @@ export function Tab({
   );
 
   return (
-    <li className={tabContainerClassNames} ref={node}>
+    <li className={tabContainerClassNames} ref={node} role="tab">
```

Comment:
In #3647 @jaxee changed this to `role="presentation"`. Which is better? I'm guessing `role=tab` is more correct?

Reply:
@BPScott It's a little more complicated than this.

The `role="tab"` is present on the [Unstyled Link / Button](https://github.com/Shopify/polaris-react/blob/master/src/components/Tabs/components/Tab/Tab.tsx#L115) - which is a child of the list element. I believe that having the `role="tab"` on the link / button to be more correct than having it on the list element. 

With this change - another a11y error will pop up saying that the button / unstyled link doesn't have a parent with the "tablist" role. 

Totally open to discussion on this 😄

Reply:
From what I understand we'll want both of them. The `ul` has a role of `tablist` so the list items are presentational (e.g role=presentation) and the anchor will have `role=tab`

Reply:
Mozilla actually uses [tabs as their example for the presentation role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role) 😄

Reply:
Awesome! My PR does exactly what Mozilla is doing 😄

Reply:
#3647 has been merged, this can be removed if you rebase atop latest master

Reply:
Awesome work gang!